docs/reference/3-admin-api/installation/admin-api-2x-for-odsapi-7x/admin-api-2x-docker-installation.md

```diff
@@ -0,0 +1,263 @@
+
+# Installation Instructions
+
+## General Prerequisites
+
+The following are required to install the Admin API:
+
+*   The Admin API provides an interface to administer an Ed-Fi ODS / API. Understandably, you must have an instance of the Ed-Fi ODS / API v7.1 deployed and operational before you can use the Admin API. Tested configurations include on-premises installation via [binary installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100419/Getting+Started+-+Binary+Installation) or [source code installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100348/Getting+Started+-+Source+Code+Installation). 
+*   A SQL Server 2012 or higher, or Postgres 11 or higher database server (i.e., the same platform requirement applicable to your ODS / API).
+*   A modern web browser such as Google Chrome, Mozilla Firefox, or Microsoft Edge is required to view live Swagger documentation. Internet Explorer 11 (a pre-installed browser on Windows Server) may load but may not function when using Admin API.
+
+# Installation Instructions
+
+Admin API is not included with the ODS-Docker solution by default, but can be hosted as part of that ecosystem.
+
+To install Admin API on Docker, first Install the [ODS / API Docker](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Docker) environment [following these instructions](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Docker+Deployment). Then, apply the below changes to the environment to introduce the Admin API.  Admin API does not support in-place upgrades from prior versions.  Please install a fresh copy of Admin API to upgrade from prior versions.
+
+## 1\. Include Admin API in the ODS Docker Setup
+
+### Docker Compose
+
+Add the following to your `docker-compose.yml`  file. This can be done either instead of or in addition to the `adminapp`  service.
+
+#### Admin API Application
+
+Docker compose
+
+> [!NOTE]
+> ![](https://edfi.atlassian.net/wiki/images/icons/grey_arrow_down.png)
+> 
+> Single-Tenant
+> 
+> ```
+> This service depends on the `pb-admin` and subsequently `db-admin` services to run.  
+> # ... above are other services adminapi: image: edfialliance/ods-admin-api:${ADMIN_API_TAG} environment: ADMIN_POSTGRES_HOST: pb-admin POSTGRES_PORT: "${PGBOUNCER_LISTEN_PORT:-6432}" POSTGRES_USER: "${POSTGRES_USER}" POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}" DATABASEENGINE: "PostgreSql" AUTHORITY: ${AUTHORITY} ISSUER_URL: ${ISSUER_URL} SIGNING_KEY: ${SIGNING_KEY} ADMIN_API_VIRTUAL_NAME: ${ADMIN_API_VIRTUAL_NAME:-adminapi} API_INTERNAL_URL: ${API_INTERNAL_URL} AppSettings__DefaultPageSizeOffset: ${PAGING_OFFSET:-0} AppSettings__DefaultPageSizeLimit: ${PAGING_LIMIT:-25} volumes: - ../../Docker/ssl:/ssl/ depends_on: - pb-admin restart: always hostname: ${ADMIN_API_VIRTUAL_NAME:-adminapi} container_name: adminapi healthcheck: test: $$ADMIN_API_HEALTHCHECK_TEST start_period: "60s" retries: 3 # ... below are network and volume configs
+> ```
```

Comment:
Ooh, something bad happened to the formatting here. I think this is supposed to be YAML, but the line breaks were lost in translation.